src/components/LoginForm.js

```diff
@@ -0,0 +1,142 @@
+import { useState, useEffect, useReducer, useContext } from "react"
+import { Link, useMatch, useResolvedPath, useNavigate } from "react-router-dom"
+import  {UserContext}  from "./UserContext.js"
+
+import axios from "axios"
+
+
+
+
+function LoginForm(){
+
+    // Boolean state to check if the user is logged in or not
+    const [logged, setLogged] = useState(false)
+    // State to store users
+    const [user, setUser] = useState({name:"", password: ""})
+    // State to store already registered users
+    const [registeredusers, setRegisteredUsers] = useState([]) 
```

Comment:
This is me being picky 😆 
```suggestion
    const [registeredUsers, setRegisteredUsers] = useState([]) 
```